compose/bin/setup-integration-tests

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+source env/db.env
+
+REAL_SRC=$(cd -P "src" && pwd)
+MYSQL_CONFIG=dev/tests/integration/etc/install-config-mysql.php
+INTEGRATION_DB=magento_integration_tests
+CREATE_DB="bin/clinotty mysqladmin -hdb -uroot -p${MYSQL_ROOT_PASSWORD} create ${INTEGRATION_DB}"  && echo "Database ${INTEGRATION_DB} created."
+bin/clinotty mysql -hdb -uroot -p${MYSQL_ROOT_PASSWORD} ${INTEGRATION_DB} -e exit &> /dev/null || $CREATE_DB
+
+sed -e "s/'db-host' => 'localhost'/'db-host' => 'db'/" \
```

Comment:
Are we ok with using root user or do we want to create a separate user?
We could put credentials in `env/db.env` if you prefer.

Reply:
I think `root` user is fine, as most of these scripts are meant for development. However, I think putting creds in `env/db.env` are the right way to go:

```
...
MYSQL_INTEGRATION_ROOT_PASSWORD=magento
MYSQL_INTEGRATION_DATABASE=magento_integration
MYSQL_INTEGRATION_USER=magento
MYSQL_INTEGRATION_PASSWORD=magento
etc
```

Note renaming `INTEGRATION_DB ` to `MYSQL_INTEGRATION_DATABASE` and moving that into the env.db file. I'd also keep all the users/creds/etc `magento` for simplicity sake.